### PR TITLE
fix(clerk-js): Correctly navigate to the continue route if needed during verification

### PR DESCRIPTION
**Pull request by desiprisg**
## Description

By adding these two new routes, we can navigate back to the `continue` step when needed, instead of going back to the start of the sign up flow.

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerkinc/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Review by panteliselef — APPROVED** [review]
💯

File: packages/clerk-js/src/ui/components/SignUp/SignUp.tsx [hunk 1]
```diff
@@ -56,7 +56,21 @@ function SignUpRoutes(): JSX.Element {
...
+            canActivate={clerk => !!clerk.client.signUp.emailAddress}
+          >
+            <SignUpVerifyEmail />
+          </Route>
+          <Route
+            path='verify-phone-number'
+            canActivate={clerk => !!clerk.client.signUp.phoneNumber}
+          >
+            <SignUpVerifyPhone />
+          </Route>
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/SignUp/SignUp.tsx:73]: ❓ Shouldn't the `index` route be the last item here?

Comment (desiprisg) [reply packages/clerk-js/src/ui/components/SignUp/SignUp.tsx:73]: Do you mean moving it as the last child? Does that matter?

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**